src/Common/src/Interop/Windows/mincore/Interop.SHGetKnownFolderPath.cs

```diff
@@ -0,0 +1,320 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        internal const int COR_E_PLATFORMNOTSUPPORTED = unchecked((int)0x80131539);
+
+        // https://msdn.microsoft.com/en-us/library/windows/desktop/bb762188.aspx
+        [DllImport(Libraries.ShellFolders, SetLastError = false, BestFitMapping = false, ExactSpelling = true)]
+        internal static extern int SHGetKnownFolderPath(
+            [MarshalAs(UnmanagedType.LPStruct)] Guid rfid,
+            uint dwFlags,
+            IntPtr hToken,
+            out string ppszPath);
```

Comment:
> In this case it makes no difference.

You would still need the UnmanagedType.LPWStr on the parameter or CharSet.Unicode on the DllImport, though, right?  Otherwise how does the runtime know to use Unicode marshaling rather than ANSI for that parameter?  Wouldn't it default to ANSI and do the wrong thing?